inc/block-patterns.php

```diff
@@ -20,6 +20,10 @@ function twentytwentytwo_register_block_patterns() {
 				'twentytwentytwo-query',
 				array( 'label' => __( 'Twenty Twenty-Two Posts', 'twentytwentytwo' ) )
 			);
+			register_block_pattern_category(
+				'twentytwentytwo-pages',
+				array( 'label' => __( 'Twenty Twenty-Two Page layouts', 'twentytwentytwo' ) )
```

Comment:
```suggestion
				array( 'label' => __( 'Twenty Twenty-Two Page Layouts', 'twentytwentytwo' ) )
```

Small note: This feels weird lowercase since no other category has a lowercase word. 😄 Let's just make it uppercase for now. 

Thanks!